boards/nxp/mr-canhubk3/nuttx-config/include/board.h

```diff
@@ -152,6 +152,18 @@
 #define PIN_LPUART1_RX    (PIN_LPUART1_RX_3 | PIN_INPUT_PULLUP)   /* PTC6 */
 #define PIN_LPUART1_TX    PIN_LPUART1_TX_3   /* PTC7 */
 
+
+/* LPUART4 /dev/ttyS3  P8B 3X7 Pin 3 Single wire RC UART */
+
+#define PIN_LPUART4_RX    0                  /* NC */
```

Comment:
This can cause problems.

Reply:
I simply don't have a LPUART4_RX pin, but that's isn't a problem for the RC UART since the TX pin is bi-directional.

Reply:
I understand that, but the question is
What port/pin will be manipulated in the open when  `s32k3xx_pinconfig(PIN_LPUART4_RX)`  is called with 0? https://github.com/PX4/NuttX/blob/px4_firmware_nuttx-10.3.0%2B/arch/arm/src/s32k3xx/s32k3xx_lowputc.c#L267

Reply:
You're right. I think the following change would work better.
https://github.com/PX4/PX4-Autopilot/blob/23ba37febebefcdef1ec3d778f5b139f606b7fef/boards/nxp/mr-canhubk3/nuttx-config/include/board.h#L158

Reply:
Yes I thought of that as well, but it is is a tad bit yuck! Could do -1, and remove the debug asserts.

Reply:
We're using all 32 bits of the UINT32_T so might have yield bad consequences then

Reply:
I was being lazy with -1, use UINT32_MAX it iwll pard out to port too high and pin to high

Reply:
Then can this even happen?
```
 if ((port >= S32K3XX_NPORTS) && (pin >= S32K3XX_NPINS))
    {
      return -EINVAL; /* Invalid port or pin number */
    }
```

Reply:
Nope that can't happen 😅